docs/contribution_guidelines.rst

```diff
@@ -35,7 +35,15 @@ Create a branch for your change
 
 .. code-block:: none
 
-  $ cd incubator-pinot
+  $ cd pinot
```

Comment:
Suggest: In line 23, last part of the sentence, add " and created your own fork of the pinot source code".